storage.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/cybozu-go/cke/server"
```

Comment:
This should be grouped separately.
If you are using `gopls` in VSCode, this is fixed automatically.

Reply:
Actually, this makes a circular reference...